diff --git a/dbt_platform_helper/utils/application.py b/dbt_platform_helper/utils/application.py
index 4611cf6..774c90d 100644
--- a/dbt_platform_helper/utils/application.py
+++ b/dbt_platform_helper/utils/application.py
@@ -33,9 +33,19 @@ class Environment:
         return self.sessions[self.account_id]
 
 
+class Service:
+    name: str
+    kind: str
+
+    def __init__(self, name: str, kind: str):
+        self.name = name
+        self.kind = kind
+
+
 class Application:
     name: str
     environments: Dict[str, Environment]
+    services: Dict[str, Service]
 
     def __init__(self, name: str):
         self.name = name
@@ -104,6 +114,19 @@ def load_application(app: str = None, default_session: Session = None) -> Applic
         ]
     }
 
+    response = ssm_client.get_parameters_by_path(
+        Path=f"/copilot/applications/{application.name}/components",
+        Recursive=False,
+        WithDecryption=False,
+    )
+
+    application.services = {
+        svc["name"]: Service(svc["name"], svc["type"])
+        for svc in [
+            json.loads(p["Value"]) for p in response["Parameters"]
+        ]
+    }
+
     return application
 
 
